src/main/java/com/assignments/questions/Reservation.java

```diff
@@ -0,0 +1,27 @@
+package com.assignments.questions;
+
+// Reservation Class - Question 2;
+public class Reservation {
+    String ticketID;
+
+    public Reservation(String ticketID){
+        this.ticketID = ticketID;
+    }
+
+    public String getTicketID(){
+        return this.ticketID;
+    }
+
+    public void setTicketID(String newTicketID){
+        this.ticketID = newTicketID;
+    }
+
+    public void showTicket(){
+        System.out.println("Your ticket ID is: " + this.ticketID);
```

Comment:
At this point of printing out the ticketID, you can just use ticketID instead of this.ticketID.  Its value has already been set in the constructor